views/fb_ads_views/derived_views/dt_fb_ads.view.lkml

```diff
@@ -15,7 +15,7 @@ sql:
         ad_insights.CampaignName  AS campaign_name,
         ad_insights.AdSetId  AS ad_set_id,
         ad_insights.AdSetName  AS ad_set_name,
-        ad_insights.meta_ads_source  AS ad_source,
+        "Meta"  AS ad_source,
```

Comment:
Why do we need this hardcoded??

Reply:
ad_insights.meta_ads_source is a synthetic column we generated for testing purposes, not part of the original Meta data transfer.

The data transfer as far as I know doesn't have a column to identify "Instagram" or "Facebook", but we need to be able to check from which source this derived table is pulling data from regardless of that. That's because Meta derived table is part of the code to generate the multisource DT: https://github.com/looker-open-source/pacing-ads-block/blob/master/views/pacing_block/multisource_ads.view.lkml

Any more doubts feel free to ask.
Thanks!